api/v1alpha3/awsmachine_types.go

```diff
@@ -73,12 +73,12 @@ type AWSMachineSpec struct {
 	// FailureDomain is the failure domain unique identifier this Machine should be attached to, as defined in Cluster API.
 	// For this infrastructure provider, the ID is equivalent to an AWS Availability Zone.
 	// If multiple subnets are matched for the availability zone, the first one return is picked.
```

Comment:
```suggestion
	// If multiple subnets are matched for the availability zone, the first one returned is picked.
```

If you could fix this as well as part of this PR, thank you!